SteamKit2/SteamKit2/Steam/SteamClient/SteamClient.cs

```diff
@@ -342,5 +368,14 @@ void HandleServerList( IPacketMsg packetMsg )
             PostCallback( new ServerListCallback( listMsg.Body ) );
         }
 
+        void HandleJobHeartbeat( IPacketMsg packetMsg )
+        {
+            jobManager.HeartbeatJob( packetMsg.TargetJobID );
+        }
+        void HandleJobFailed( IPacketMsg packetMsg )
```

Comment:
There's #171 regarding this, but I'm not sure if consumers really want to go through the code gymnastics of handling failures (and likely heartbeats) by having multiple callbacks.